openedx/core/djangoapps/course_groups/views.py

```diff
@@ -431,9 +479,9 @@ def debug_cohort_mgmt(request, course_key_string):
 
 @expect_json
 @login_required
-def cohort_discussion_topics(request, course_key_string):
+def divided_discussion_topics(request, course_key_string):
     """
-    The handler for cohort discussion categories requests.
+    The handler for divided discussion categories requests.
```

Comment:
Yeah, it looks like this returns all discussion topics, so I would remove "divided" from the name.